Maddori.Apple/Maddori.Apple/Screen/InProgress/InProgressViewController.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  InProgressViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/24.
+//
+
+import UIKit
+
+import SnapKit
+
+final class InProgressViewController: BaseViewController {
+    
+    private var keywordData = Keyword.mockData
+    private let currentRetrospectiveUser = "진저"
+//    private let user = "이드"
+    private let user = "진저"
+    private enum Size {
+        static let keywordLabelHeight: CGFloat = 50
+        static let sectionPadding: CGFloat = 60
+    }
+    private var keywordsSectionList = [[Keyword]]()
```

Comment:
```suggestion
    private var keywordsSectionList: [[Keyword]] = []
```

p5;

진짜 컨벤션에 집착하는 이상한 사람같아 보일 수도 있는데...
위와 같이 수정하는 건 어떨까요??
노션의 컨벤션에 적어두진 않았지만, mockData도 그렇고, baseViewController 에서도 그렇고 `var sampleArray: [] = []`의 형식으로 사용해왔기 때문입니다... 하하 (머쓱)

Reply:
[89aa89f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/59/commits/89aa89fbc0a6bd6a65f1a42cd8d38da91e8cdc19)
반영했습니다!!